README.md

```diff
@@ -42,9 +42,8 @@ Here are some commands you can run:
 | yarn amo:dev-https | Same as `amo:dev` but with HTTPS, available at: https://example.com:3000/. [Read about setting up this environment](docs/moz-addon-manager.md#developing-with-a-local-https-server-recommended) |
 | yarn amo:no-proxy | Start the dev server without a proxy (for amo) using data from Docker |
 | yarn amo:stage | Start the dev server/proxy (for amo) using data from the staging server (https://addons.allizom.org/) |
-| yarn build | Build an app specified with the `NODE_APP_INSTANCE` environment variable. |
-| yarn build-all | Build all the applications. |
-| yarn build-ci | Run the `build-all` and `bundlesize` npm scripts. |
+| yarn build | Build the app. |
+| yarn build-ci | Run the `build` and `bundlesize` npm scripts. |
```

Comment:
Grep for "NODE_APP_INSTANCE" in the rest of the files, I see some other references to this env var on L290, L302 and L303

Reply:
Yeah, there were some in some `.md` files. I've updated those in the next commit.

Reply:
in which commit? (the current patch still has some references to the env var)

Reply:
In [`ddf8710` (#9830)](https://github.com/mozilla/addons-frontend/pull/9830/commits/ddf87100fa384c68833c821d57eebb11420bb9d0). You keep checking before I've pushed the next commit!

Reply:
🤣